### ISSUE TITLE
#382: oneliners

### ISSUE THREAD

**Issue by uweschaefer** [factcast/factcast#383]
_From @uweschaefer on 2019-04-14 19:54:37 +0000 UTC_

_Copied from original issue: Mercateo/factcast#383_

**Comment by uweschaefer** [comment]
_From @codecov[bot] on 2019-04-14 19:57:50 +0000 UTC_

# [Codecov](https://codecov.io/gh/Mercateo/factcast/pull/383?src=pr&el=h1) Report
> Merging [#383](https://codecov.io/gh/Mercateo/factcast/pull/383?src=pr&el=desc) into [master](https://codecov.io/gh/Mercateo/factcast/commit/6fde21196caf2f138edd1d2db6347006a6554bdb?src=pr&el=desc) will **not change** coverage.
> The diff coverage is `n/a`.

[![Impacted file tree graph](https://codecov.io/gh/Mercateo/factcast/pull/383/graphs/tree.svg?width=650&token=Kt03N0Cztt&height=150&src=pr)](https://codecov.io/gh/Mercateo/factcast/pull/383?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##           master     #383   +/-   ##
=======================================
  Coverage   91.84%   91.84%           
=======================================
  Files          70       70           
  Lines        1680     1680           
  Branches      150      150           
=======================================
  Hits         1543     1543           
  Misses         73       73           
  Partials       64       64
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/Mercateo/factcast/pull/383?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/Mercateo/factcast/pull/383?src=pr&el=footer). Last update [6fde211...c85ce39](https://codecov.io/gh/Mercateo/factcast/pull/383?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by uweschaefer** [comment]
_From @uweschaefer on 2019-04-14 19:59:12 +0000 UTC_

closes #382
